### DISCUSSION TITLE
Working setup

### DISCUSSION THREAD

**Discussion by Caleb9** [Caleb9/asuswrt-merlin-nordvpn-wg-updater#1] [General]
Please, if you went through the effort of setting the script up successfully, report the router model and Merlin firmware version. Thanks!

**Comment by carmenfdezb** [comment]


**Reply by Caleb9** [reply]
Hi @carmenfdezb, please see [this](https://github.com/Caleb9/asuswrt-merlin-nordvpn-wg-updater/issues/2#issuecomment-2348147078) and let me know if it helped. Thanks.

**Reply by Caleb9** [reply]
I updated the script to fix the issue. You can also try the new version.

**Comment by morlan** [comment]
Working on RT-AX88U with Merlin 3004.388.8_2

**Comment by Caleb9** [comment]
Merlin 3006.102.4 on RT-AX86U Pro works.
